ansible_base/authentication/models/authenticator_user.py

```diff
@@ -50,3 +55,6 @@ def save(self, *args, **kwargs):
         self.extra_data = b64_encode_binary_data_in_dict(self.extra_data)
 
         super().save(*args, **kwargs)
+
+        # After super().save extra_data will be encrypted but we need this to field to not be encrypted during the login process
+        self.extra_data = json.loads(ansible_encryption.decrypt_string(self.extra_data))
```

Comment:
We shouldn't need json.loads here anymore if we are going to do it inside of the decrypt_string function.